src/main/java/com/rcv/TabulatorSession.java

```diff
@@ -55,12 +55,35 @@
   // param: configPath path to config json file
   TabulatorSession(String configPath) {
     this.configPath = configPath;
+    // current date-time formatted as a string used for creating unique output files names
+    timestampString = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss").format(new Date());
   }
 
   String getTimestampString() {
     return timestampString;
   }
 
+  // purpose: special mode to just export the CVR as CDF JSON instead of tabulating
+  void convertToCdf() {
+    ContestConfig config = ContestConfig.loadContestConfig(configPath);
+    if (config != null && config.validate()) {
```

Comment:
validate is done for you by loadContestConfig

Reply:
@tarheel I'm going to push a couple changes to this branch: 
merging in master to get the CDFReader
small fix to the config files (CDFReader expects an empty candidates list).
There's still the old json formatting issue where where you ran the formatting code on expected json files and it doesn't match, but I think it's close now.

Reply:
oh... actually i think there are 2 problems (maybe I'm missing something)
1 no way to indicate cdf output in a config file on disk (only through ui which will not work with tests)
2 no way to indicate in tests that cdf output file should be compared (currently test always compares to summary json)

Reply:
> validate is done for you by loadContestConfig

That method validates the RawContestConfig, but not the ContestConfig (which is a different form of validation).

Reply:
Ah you are right.  I don't even know my own validate code I just worked on :(